src/Components/Publishing/ToolTip/ArtistToolTip.tsx

```diff
@@ -1,60 +1,29 @@
 import styled from "styled-components"
 import React from "react"
 import { map } from "lodash"
+import { createFragmentContainer, graphql } from "react-relay"
 import { garamond, unica } from "Assets/Fonts"
 import fillwidthDimensions from "../../../Utils/fillwidth"
 import { ToolTipDescription } from "./Components/Description"
 import { FollowButton } from "./Components/FollowButton"
 import { NewFeature } from "./Components/NewFeature"
 import { ArtistMarketData } from "./Components/ArtistMarketData"
+import { ArtistToolTip_artist } from "../../../__generated__/ArtistToolTip_artist.graphql"
 
-export interface ArtistProps {
-  auctionResults?: {
-    edges: Array<{
-      node: {
-        price_realized: {
-          display: string
-        }
-      }
-    }>
-  } | null
-  blurb?: string
-  collections?: string[]
-  formatted_nationality_and_birthday?: string
-  highlights?: {
-    partners?: {
-      edges?: Array<{
-        node?: {
-          categories: Array<{
-            id: string
-          }>
-        }
-      }>
-    } | null
-  } | null
-  href?: string
-  carousel?: {
-    images?: Array<{
-      resized: {
-        url: string
-        width: number
-        height: number
-      }
-    }>
-  }
-  name?: string
+export interface ArtistToolTipProps {
   showMarketData?: boolean
+  artist: ArtistToolTip_artist
```

Comment:
Why are you exporting this as part of the interface?

Reply:
It's being used to type check/validate the shape of Fixtures that are used in storybooks and tests (https://github.com/artsy/reaction/pull/664/files?utf8=%E2%9C%93&diff=unified#diff-e9af1869999b3d835b6944bbbfe05a8cR660)